README.md

```diff
@@ -1,57 +1,16 @@
-# Northcoders House of Games API
+1. Copy Trello Board - Complete
 
-## Background
+2. Project Setup:
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+Initial Set up:
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+To Clone Repository: Git Clone https://github.com/Souness1982/Stephen-Souness---Back-End-Project.git
 
-## Kanban
+npm install
```

Comment:
we only need to tell the user to npm install here because you've installed all the dependencies necessary for them already, they will be stored in the package.json